openedx/core/djangoapps/agreements/views.py

```diff
@@ -119,3 +121,43 @@ def post(self, request, course_id):
         signature = create_integrity_signature(username, course_id)
         serializer = IntegritySignatureSerializer(signature)
         return Response(serializer.data)
+
+
+class LTIPIISignatureView(AuthenticatedAPIView):
+    """
+    Endpoint for a LTI PII Signature
+    /lti_pii_signature/{course_id}
+
+    HTTP POST
+        * Do not create an LTIPIISignature record if the feature is not enabled via the CourseWaffleFlag
+        * If an LTI PII signature does not exist for the user + course, creates one and
+          returns it. If one does exist, returns the existing signature.
+    """
+
+    def post(self, request, course_id):
+        """
+        Create an LTI PII signature for the requesting user and course. If a signature
+        already exists, returns the existing signature instead of creating a new one.
+
+        /api/agreements/v1/lti_pii_signature/{course_id}
+
+        Example response:
+            {
+                username: "janedoe",
+                course_id: "org.2/course_2/Run_2",
+                created_at: "2021-04-23T18:25:43.511Z"
+            }
+        """
+        if not settings.FEATURES.get('ENABLE_LTI_PII_ACKNOWLEDGEMENT'):
```

Comment:
We had originally planned to use the `CourseWaffleFlag` you added in https://github.com/openedx/edx-platform/pull/33437 here. I discussed this with Spencer, and we can keep this Django setting instead. I can log a ticket to remove the code you added in https://github.com/openedx/edx-platform/pull/33437. But I wanted to check that you're on board with that before I do that. Let me know!

Reply:
Also, if we're going to use this Django setting, we should set a default value of `False` in the `lms/envs/common.py` and `cms/env/common.py` settings files to be explicit that this feature is currently not enabled. You'll also want to add documentation for this toggle to both files. Here is an [example](https://github.com/openedx/edx-platform/blob/a95809763068f13477759a4f1eddfd7e3e203a36/lms/envs/common.py#L962-L971) of the integrity signature documentation in the `lms/envs/common.py` file.

Reply:
Yes it is okay to remove the code I wrote. Thank you for letting me know about the changes. I removed the lines mentioning the waffle flag.